usaspending_api/database_scripts/etl/transaction_delta_view.sql

```diff
@@ -30,8 +30,17 @@ SELECT
 
   UTM.product_or_service_code,
   UTM.product_or_service_description,
+  CASE
+    WHEN UTM.product_or_service_code IS NOT NULL THEN CONCAT('{"code":"', UTM.product_or_service_code, '","description":"', NULL, '"}')
```

Comment:
Shouldn't the description have a value of `UTM.product_or_service_description` instead of `NULL`?

Reply:
probably